ci/scripts/r_deps.sh

```diff
@@ -25,8 +25,8 @@ source_dir=${1}/r
 pushd ${source_dir}
 
 # Install R package dependencies
-${R_BIN} -e "install.packages(c('remotes', 'dplyr', 'glue'))"
+${R_BIN} -e "install.packages('remotes'); remotes::install_cran(c('glue', 'rcmdcheck'))"
 ${R_BIN} -e "remotes::install_deps(dependencies = TRUE)"
-${R_BIN} -e "remotes::install_github('romainfrancois/decor')"
+${R_BIN} -e "remotes::install_github('nealrichardson/decor')"
```

Comment:
Should we wait for https://github.com/romainfrancois/decor/pull/1 ?

Reply:
I wouldn't let this hold up the PR. I'm happy to fold that change into a future patch, once Romain merges that.

An alternative would be to remove `decor` entirely because it's not used in CI; it's just installed here as convenience in case a (hypothetical) developer wanted to pull the docker image and edit the Rcpp code.

Reply:
OK.